cmd/mysterium_client/command_run/command.go

```diff
@@ -71,7 +74,9 @@ func (cmd *CommandRun) Run(options CommandOptions) (err error) {
 		return err
 	}
 
-	apiEndpoints := tequilapi.NewApiEndpoints()
+	idm := identity.NewIdentityManager("TEST")
```

Comment:
This password should be input for identity create method, e.g. it's optional for listing identities